python/mlx/nn/layers/base.py

```diff
@@ -159,6 +159,9 @@ def load_weights(
               only the weights actually contained in the model are loaded and
               shapes are not checked. Default: ``True``.
 
+        Returns:
+            The model with the updated weights.
```

Comment:
To further reduce the risk of confusion perhaps we can reword this to "The module instance after updating the weights", just to reinforce that it is the same module being returned.

Reply:
Sure. Thanks for the comment!